addon/nodes/block-rdfa.ts

```diff
@@ -18,6 +17,6 @@ export const block_rdfa: NodeSpec = {
     },
   ],
   toDOM(node: PNode) {
-    return [node.attrs.__tag, node.attrs, 0];
+    return ['div', node.attrs, 0];
```

Comment:
I'm not sure about that, as the `__tag` attribute was never parsed in the `parseDOM` rules and is also never set manually, so it would always have been its default value (`div`).

Reply:
ah fair enough